OrbitGl/ProcessManager.h

```diff
@@ -39,8 +41,11 @@ class ProcessManager {
   virtual void SetProcessListUpdateListener(
       const std::function<void(ProcessManager*)>& listener) = 0;
 
+  virtual outcome::result<std::vector<ModuleInfo>, std::string> GetModuleList(
```

Comment:
Maybe the naming of this function and `process_list()` can be improved to signal which ones are blocking (aka should not be run on the main thread). Maybe something like `GetModuleListFromRemote` or maybe just a comment.

Reply:
I changed the naming convention. ptal.

Reply:
I like that with using Load and Get, will change my PR accordingly